hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -154,6 +123,19 @@ read_env constraints.sdc.env
 touch $USER_ENV_FILE
 read_env $USER_ENV_FILE
 
+# RI's recommended approach to filter out unnecessary errors on async_fifo
+catch {rm tmp.env}
+set outfile [open tmp.env  w]
+puts $outfile "set_data_clock_domain rdata_o -module prim_fifo_async -derived_from rvalid_o"
+# TODO: These should not be hardcoded here, instead, we need to create another variable called CDC_CONSTRAINT
+# where a top could specifically suppli this
+puts $outfile "set_stable {{top_earlgrey.u_pinmux_aon.mio_oe_retreg_q[46:0]}}"
+puts $outfile "set_stable {{top_earlgrey.u_pinmux_aon.dio_out_retreg_q[15:0]}}"
+puts $outfile "set_clock_sense -stop_propagation -clocks { AST_EXT_CLK } { u_ast.clk_ast_ext_i }"
+
+close $outfile
```

Comment:
something for later, but is there a need to actually put these constraints into a file and then read that file back? Why not invoke the commands directly in this file? 

we can address that later, let's merge this for now!

Reply:
@tjaychen thanks. Yeah, as I replied to the other comment above, this will be addressed later.